src/utils/checkFileExists.ts

```diff
@@ -0,0 +1,9 @@
+import { existsSync } from "fs";
+import path from 'path';
+
+export const checkFileExists = (resolutionPath: string) => {
+  if (existsSync(path.resolve(__dirname, resolutionPath))) {
```

Comment:
we have a similar logic in `src/utils/createFile.ts` I'd recommend extracting that into own util and not using the `sync` version:

```
import { promisify } from 'util';
import {exists} from 'fs'

const existFile = (path: string) => {
  return promisify(fs.exists)(path);
}
```